apps/sms/services/test/unit/moz_mobile_message/messaging_service_test.js

```diff
@@ -0,0 +1,670 @@
+/*global bridge,
+         MocksHelper
+*/
+
+'use strict';
+
+require('/services/js/bridge_service_mixin.js');
+require('/services/test/unit/mock_bridge.js');
+require('/views/shared/test/unit/mock_messages.js');
+require('/views/shared/test/unit/mock_settings.js');
+require('/views/shared/test/unit/mock_smil.js');
+
+var MocksHelperForAttachment = new MocksHelper([
+  'bridge',
+  'Settings',
+  'SMIL'
+]).init();
+
+var MessagingService, successEvt, errorEvt;
+
+suite('Messaging service >', function() {
+
+  MocksHelperForAttachment.attachTestHelpers();
+
+  suiteSetup(function(done) {
+    window.self = {};
+    window.importScripts = sinon.stub();
+
+    require('/services/js/moz_mobile_message/messaging_service.js', done);
+  });
+
+  suiteTeardown(function() {
+    delete window.self;
+    delete window.importScripts;
+  });
+
+  setup(function() {
+    MessagingService =self.MessagingService;
+    successEvt = {
+      target: {
+        result: {}
+      }
+    };
+    errorEvt = {
+      target: {
+        error: new Error()
+      }
+    };
+  });
+
+  suite('init', function() {
+    var serviceStub;
+
+    setup(function() {
+      serviceStub = sinon.stub({
+        method: () => {},
+        stream: () => {},
+        broadcast: () => {}
+      });
+
+      sinon.stub(bridge, 'service').returns(serviceStub);
+      sinon.stub(bridge, 'client');
+      MessagingService.init();
+    });
+
+    test('service mixing and client are initialized', function() {
+      sinon.assert.called(importScripts, '/lib/bridge.js');
```

Comment:
just a guess: currently ```setup``` is failing, so there is a chance that this test just isn't run :)